src/styles/_catalog.scss

```diff
@@ -1,10 +1,28 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
+  display: grid;
+  grid-template-columns: repeat(auto-fill, 200px);
+  justify-content: center;
+  gap: 48px;
 
   max-width: 944px;
   margin: 0 auto;
```

Comment:
If u set repeat (4, 200px) - ur page will be with 4 cards, so u don't need max-width and margin

Reply:
and you can't use margin for BEM block, remove it

Reply:
ooops, sry, stupid mistake ;)